tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot

```diff
@@ -242,7 +226,7 @@ Connectivity Bridge to Management
     Should Not Contain  ${output}  Error:
 
     Log To Console  Creating bridge container
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d --net=bridge --name b1 ${busybox} /bin/top
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d --net=%{BRIDGE_NETWORK} --name b1 ${busybox} /bin/top
```

Comment:
All containers in this configuration **must** be run with `-d` and `docker logs` used to get the container output if that is needed. As written this means changing the `docker attach` to a `docker logs` - which it should be anyway given attach in this manner does not guarantee we get all of the output.